ofrak_core/ofrak/service/resource_service.py

```diff
@@ -33,8 +33,8 @@
 
 
 class LowValue:
-    def __lt__(self, other):
-        return True
+    def __gt__(self, other):
+        return False
```

Comment:
```suggestion
    def __gt__(self, other):
        return False
```

```suggestion
    def __lt__(self, other):
        return True
```

I think we should keep this to `__lt__` as that is the default comparison method, and `<` is the only operator used in `bisect_left`/`bisect_right` where `LowValue` is used. Is there a compelling reason to change this?

Reply:
Function coverage revealed that the `__lt__` function never gets run, which is why I switched it.

`LOW_VALUE` is used as such:
```
min_index = bisect.bisect_left(self.index, (self.min, LOW_VALUE))
```

Inside of `bisect_left` this ends up being:
```
a: ResourceNode
b: LowValue

a < b
```

Since `ResourceNode` was:
```
    def __lt__(self, other):
        if not isinstance(other, ResourceNode):
            return False
        return self.model.id < other.model.id
```

The `LowValue.__lt__` was never evaluated. This PR changes `ResourceNode.__lt__` to:

```
    def __lt__(self, other):
        if not isinstance(other, ResourceNode):
            return NotImplemented
        return self.model.id < other.model.id
```

Returning `NotImplemented` changes the expression Python evaluates:
```
a < b
```

to:

```
b > a
```

since Python falls back to the the reflected operation (https://docs.python.org/3/library/constants.html#NotImplemented).

This is why I changed it from `__lt__` to __gt__`

Reply:
> Returning NotImplemented changes the expression Python evaluates:
> a < b
> to:
> b > a

Woah! That's kind of cool. Makes sense.